code/modules/surgery/organs/ears.dm

```diff
@@ -43,10 +38,10 @@
 
 /obj/item/organ/internal/ears/proc/RestoreEars()
 	deaf = 0
-	ear_damage = 0
+	damage = 0
 
 /obj/item/organ/internal/ears/proc/AdjustEarDamage(ddmg, ddeaf)
```

Comment:
As you're moving away from this ear damage system, you should do away with `AdjustEarDamage` and `RestoreEars` entirely, and instead replace all their instances with typical organ damage code to keep it standardized. They seem to be fairly snowflaked procs for handling this weird edge case of organ damage. Heads up, there are two different procs for it under `/mob/living/carbon` and `/obj/item/organ/internal/ears` (: 

I think you'll probably want to replace the uses with procs like `E.heal_internal_damage()`, `E.receive_damage()`, and move the deafness stuff into respective overrides.